Airports/EGCN/Sids.txt

```diff
@@ -1,6 +1,8 @@
-SID:EGCN:20:ROGA20S:05ADM 15DME 35DME D322B ROGAG
-SID:EGCN:20:ROGA20N:05ADM 25DME D015K LAMIX ROGAG
-SID:EGCN:02:ROGAG02:DME01 D017E ROGAG
```

Comment:
You can remove DME01, D017E, 05ADM, 15DME, 35DME,  D322B

From Navaids/Fixes_SIDs_STARs.txt

Reply:
Fixed